pkg/services/mysqldb/provision.go

```diff
@@ -30,6 +32,39 @@ func (s *serviceManager) ValidateProvisioningParameters(
 			fmt.Sprintf(`invalid sslEnforcement option: "%s"`, pp.SSLEnforcement),
 		)
 	}
+
+	startIP := net.ParseIP(pp.FirewallIPStart)
+	if pp.FirewallIPStart != "" && net.ParseIP(pp.FirewallIPStart) == nil {
```

Comment:
Suggestion-- don't do `net.ParseIP(pp.FirewallIPStart)`, because you've already done it above and assigned it to the `startIP` variable. Change this line to:

```
if pp.FirewallIPStart != "" && startIP == nil {
```

Reply:
Yeah sorry! Was refactoring that and forgot to replace it. Thanks!